lib/internal/Magento/Framework/Stdlib/Cookie/PhpCookieManager.php

```diff
@@ -20,26 +22,32 @@
  * sensitive data so that extra protection can be added to the contents of the cookie as well as how the browser
  * stores the cookie.
  *
+ * RFC 2109 - Page 15
+ * http://www.ietf.org/rfc/rfc6265.txt
+ *
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  */
 class PhpCookieManager implements CookieManagerInterface
 {
-    /**#@+
-     * Constants for Cookie manager.
-     * RFC 2109 - Page 15
-     * http://www.ietf.org/rfc/rfc6265.txt
+    /**
+     * Max cookie size
      */
-    const MAX_NUM_COOKIES = 50;
```

Comment:
Removing a constant is a backwards incompatible change, since some third-party customization could rely on it. Please revert it and add a deprecated annotation.